views.go

```diff
@@ -21,6 +24,30 @@ import (
 
 const maxLogBytes = 4 * 1024 * 1024 // 4 MB
 
+const (
+	corsOriginsEnvVariable = "LK_CORS_ORIGINS"
+)
+
+func getCorsOrigins() []string {
+	origins := os.Getenv(corsOriginsEnvVariable)
+	if origins == "" {
```

Comment:
I was just wondering about this.
[It looks like](https://pkg.go.dev/strings#Split) it will return "a slice of length 1 whose only element is" the empty string. Maybe? ([playground](https://go.dev/play/p/1FxLUg83DwC))

Reply:
In that case maybe i should revert  my commit?

Reply:
I mean, I think the empty string won't match anything. 
But it does seem like it would make it harder to understand what's happening.